app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -90,9 +91,20 @@ def refresh_targets_for_ems(ems, targets)
 
           Benchmark.realtime_block(:save_inventory) { save_inventory(ems, target, hashes) }
           _log.info "#{log_header} Refreshing target #{target.class} [#{target.name}] id [#{target.id}]...Complete"
+
+          if hashes.kind_of?(Array)
```

Comment:
I think we should at least be sending a `ManagerRefresh::Inventory::Persister` instead of an `Array` here so it is more obvious what case we are dealing with, but that can be a follow-up refactoring PR lets just not lose track of it